Assets/Scripts/PositionTracker.cs

```diff
@@ -52,9 +52,15 @@ private void OnTriggerEnter(Collider other)
 
             Bullseye bullseyeObj = other.GetComponent<Bullseye>();
 
+            Rigidbody rb = GetComponent<Rigidbody>();
```

Comment:
you should probably just have the requires component attribute at the top of the class instead of making the rigidbody check but this will also work fine